src/Nethermind/Nethermind.Mev/Source/CompareMevBundleByIdentity.cs

```diff
@@ -15,12 +15,22 @@
 //  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
 // 
 
+using System.Collections.Generic;
 using Nethermind.Mev.Data;
 
 namespace Nethermind.Mev.Source
 {
-    public interface ISimulatedBundleFilter
+    public class CompareMevBundleByIdentity : IComparer<MevBundle>
     {
```

Comment:
is it needed?

Reply:
EqualityComparer?

Reply:
yes SortedPool needs to be able to compare by some identity.